tests/test_click.py

```diff
@@ -68,4 +68,3 @@ def test_kerberos_works():
         ],
     )

```

Comment:
You need tests for most popular hashes :-) Call both the regex and click module to make sure it works 100%

Reply:
👍🏻